docs/rc.xml.all

```diff
@@ -132,6 +132,8 @@
     <!-- Let client redraw its contents while resizing -->
     <drawContents>yes</drawContents>
     <cornerRange>8</cornerRange>
+    <!-- Borders are effectively 8 pixels wide regardless of visual appearance -->
+    <minimumArea>8</minimumArea>
```

Comment:
Missed that in the earlier PR, `<cornerRange>` should be removed and potentially replaced with a comment about it defaulting to half of titlebar size. Not sure about the comment though.

Reply:
Good catch. I replaced the stale entry with a comment explaining its use.